tests/models/arith/arith7.optimize.expected

```diff
@@ -1,5 +1,7 @@
 
 unknown
+; This model is a best-effort. It includes symbols
+; for which model generation is known to be incomplete.
```

Comment:
Which ones? This problem only uses the builtin `<` and `<=` on reals, as far as I can tell.

Reply:
I didn't fix this issue because I don't understand your point as I explained here: https://github.com/OCamlPro/alt-ergo/pull/925#discussion_r1418918197

Reply:
I replied there (sorry, it got lost int the notifications somewhere) but I don't think this is related; `arith7` doesn't look like it is using any suspicious symbols.